modules/cli/src/test/java/org/apache/ignite/internal/cli/commands/flow/FlowTest.java

```diff
@@ -53,6 +55,25 @@ class FlowTest {
     private StringWriter out;
     private StringWriter errOut;
 
+    @Test
+    void testVerbose() throws IOException {
```

Comment:
this test is good and it would be nice to test CallExcecutionPipeline too. Maybe it could be just a regular command test but with --verbose flag.